scheduler/src/cook/mesos/schema.clj

```diff
@@ -315,6 +315,32 @@ for a job. E.g. {:resources {:cpus 4 :mem 3} :constraints {\"unique_host_constra
              any semantic meaning other than they will be considered schedulable
              at the same time. A primitive of job group will be added to add
              more semantic power in the future."}
+   ; compute-cluster types
+   {:db/id (d/tempid :db.part/user)
+    :db/ident :compute-cluster.type/mesos}
+   {:db/id (d/tempid :db.part/user)
+    :db/ident :compute-cluster.type/kubernetes}
+   ;; compute-cluster
+   {:db/id (d/tempid :db.part/db)
+    :db/ident :compute-cluster/type
+    :db/valueType :db.type/ref
+    :db/cardinality :db.cardinality/one
+    :db.install/_attribute :db.part/db}
+   {:db/id (d/tempid :db.part/db)
```

Comment:
I would vote to make this `:db/unique :db.unique/identity`. That's going to clean up a lot of the query code and remove the need to store db ids everywhere.